examples/flutter_gallery/lib/demo/material/chip_demo.dart

```diff
@@ -297,7 +309,7 @@ class _ChipDemoState extends State<ChipDemo> {
                 _showShapeBorder = !_showShapeBorder;
               });
             },
-            icon: const Icon(Icons.vignette),
+            icon: const Icon(Icons.vignette, semanticLabel: 'Update border shape'),
```

Comment:
Could you add a test to make sure the label gets set?

Reply:
Done